_posts/de/newsletters/2022-10-26-newsletter.md

```diff
@@ -310,13 +310,13 @@ Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
   benötigt, bevor es verwendet werden kann.
 
 - [Eclair #2463][] und [#2461][eclair #2461] aktualisieren die Eclair-
-  Implementierung der [interaktiven und dualen Finanzierungsprotokolle][topic
-  dual funding], welche erfordert, dass sich jeder Finanzierungsinput für
-  [RBF][topic rbf] anmeldet und auch bestätigt wird (d.h. einen Output ausgibt,
-  der sich bereits in der Blockchain befindet). Diese Änderungen stellen sicher,
-  dass RBF verwendet werden kann und dass keine der Gebühren, die ein
-  Eclair-Benutzer beisteuert, dazu verwendet wird, frühere Transaktionen seiner
-  Peers zu bestätigen.
+  Implementierung der [interaktiven und beidseitigen Finanzierungsprotokolle]
+  [topic dual funding], welche erfordert, dass sich jeder Finanzierungsinput als
+  ersetzbar ([RBF][topic rbf])  gekennzeichnet wird und eine bereits bestätigte
+  Ausgabe umsetzt (d.h. die Parent-Transaktion sich bereits in der Blockchain
+  befindet). Diese Änderungen stellen sicher, dass RBF verwendet werden kann und
+  dass keine der Gebühren, die ein Eclair-Benutzer beisteuert, dazu verwendet
+  wird, frühere Transaktionen seiner Peers zu bestätigen.
```

Comment:
Just a grammar nit, looks good to go otherwise:

```suggestion
  dass keine der Gebühren, die ein Eclair-Benutzer beisteuert, dazu verwendet
  werden, frühere Transaktionen seiner Peers zu bestätigen.
```